scala/scala_toolchain.bzl

```diff
@@ -1,13 +1,21 @@
+load(
+    "@io_bazel_rules_scala//scala:providers.bzl",
+    _ScalacProvider = "ScalacProvider")
+
 def _scala_toolchain_impl(ctx):
   toolchain = platform_common.ToolchainInfo(
       scalacopts = ctx.attr.scalacopts,
+      scalac_repositories = ctx.attr.scalac_repositories,
       unused_dependency_checker_mode = ctx.attr.unused_dependency_checker_mode)
   return [toolchain]
 
 scala_toolchain = rule(
     _scala_toolchain_impl,
     attrs = {
         'scalacopts': attr.string_list(),
+        'scalac_repositories': attr.label(
+            default = "@io_bazel_rules_scala//scala:scala_repositories_default",
```

Comment:
Interesting... if we are back to putting the repositories into the toolchain, which I actually like, since it seems closer to what bazel devs expect, can we not remove it entirely and put all the things the provider give directly on the toolchain?